app/scss/vendor/font-awesome/less/path.less

```diff
@@ -0,0 +1,15 @@
+/* FONT PATH
```

Comment:
In the future try and build font awesome using the new technique i show yall at the end of the class.
that way you wont have to include vender sources in you version controll